src/libs/actions/SignInRedirect.js

```diff
@@ -49,10 +55,28 @@ function clearStorageAndRedirect(errorMessage) {
         });
 }
 
+/**
+ * Reset all current params of the Home route
+ */
+function resetHomeRouteParams() {
+    Navigation.isNavigationReady().then(() => {
+        const routes = navigationRef.current && lodashGet(navigationRef.current.getState(), 'routes');
+        const homeRoute = lodashFind(routes, route => route.name === SCREENS.HOME);
```

Comment:
```suggestion
        const homeRoute = _.find(routes, route => route.name === SCREENS.HOME);
```
Nitpick but can we use `_.find()` instead here? Seems to be our go-to throughout the App